origin-growth/src/resources/invite.js

```diff
@@ -55,6 +55,50 @@ class GrowthInvite {
     })
   }
 
+  /**
+   * Creates a referrer - referee connection in case this referee does not have a referrer yet
+   *
+   * @param {string} code - growth invitation code
+   */
+  static async makeReferralConnection(code, walletAddress) {
+    try {
+      const referralLink = await db.GrowthReferral.findOne({
+        where: {
+          refereeEthAddress: walletAddress
```

Comment:
to be safe, do the look up lowercase in case it gets passed checksum.

> refereeEthAddress: walletAddress.toLowerCase()

Reply:
👍